StepLang.Wiki/TYP005.md

```diff
@@ -0,0 +1,37 @@
+# Variable Already Declared Exception
+
+## Description
+
+This exception is thrown when a variable is declared more than once in the same scope.
+
+Example:
+
+```step
+number a = 1
+
+number a = 2
+```
+
+## Remediation
+
+Make sure that each variable is only declared once in each scope or give each variable a different name.
```

Comment:
maybe: "or, re-assign the existing variable to a different value if you no longer need the old value"

Reply:
Added another code block to show this usage.